src/index.html

```diff
@@ -13,6 +15,53 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    
+    <div class="stars stars--0">
```

Comment:
There should be an empty line between sibling elements to add some 'air' and simplify reading, as per the checklist.